pkg/api-server/resource_endpoints.go

```diff
@@ -532,6 +532,13 @@ func (r *resourceEndpoints) validateLabels(rm model.ResourceMeta) validators.Val
 		}
 	}
 
+	if _, ok := rm.GetLabels()[mesh_proto.PolicyRoleLabel]; ok {
+		err.AddViolationAt(validators.Root().Key(mesh_proto.PolicyRoleLabel), fmt.Sprintf("%s label should not be set manually", mesh_proto.PolicyRoleLabel))
```

Comment:
should be the same as on k8s, we allow these labels to be set, but validate the value to be correct

Reply:
right, I've forgot to update it after fixing k8s validation